File: Exemplos/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Java-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/BRIDGE/ImprimirCupomSat.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.BRIDGE

import com.elgin.intent_digital_hub.IntentDigitalHubService.BRIDGE.BridgeCommand

class ImprimirCupomSat(private val xml: String) : BridgeCommand("ImprimirCupomSat") {
    override fun functionParameters(): String {
        return "\"xml\"" + ":" + "\"" + xml + "\""
    }
}

File: Exemplos M10/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/BRIDGE/imprimir_cupom_sat.dart
import 'package:intent_digital_hub/IntentDigitalHubService/BRIDGE/bridge_command.dart';

class ImprimirCupomSat extends BridgeCommand {
  final String _xml;

  ImprimirCupomSat(this._xml) : super("ImprimirCupomSat");

  @override
  functionParameters() => '"xml"' ':' '"' '$_xml' '"';
}
